File: CS/CustomWizardExample/Wizard/Presenters/CustomChooseDataSourceNamePage.cs
using System;
using DevExpress.DataAccess.Wizard.Model;
using DevExpress.DataAccess.Wizard.Views;
using DevExpress.DataAccess.Wizard.Presenters;
using DevExpress.DataAccess.Wizard.Services;

namespace CustomWizardExample.Wizard.Presenters {
    public class CustomChooseDataSourceNamePage : ChooseDataSourceNamePage<DataSourceModel> {
        public CustomChooseDataSourceNamePage(IChooseDataSourceNamePageView view, IDataSourceNameCreationService dataSourceNameCreator) 
            : base(view, dataSourceNameCreator) { }
        public override Type GetNextPageType() {
            return typeof(CustomChooseDataSourceTypePage);
        }
    }
}

File: VB/CustomWizardExample/Wizard/Presenters/CustomChooseDataSourceNamePage.vb
Imports System
Imports DevExpress.DataAccess.Wizard.Model
Imports DevExpress.DataAccess.Wizard.Views
Imports DevExpress.DataAccess.Wizard.Presenters
Imports DevExpress.DataAccess.Wizard.Services

Namespace CustomWizardExample.Wizard.Presenters
    Public Class CustomChooseDataSourceNamePage
        Inherits ChooseDataSourceNamePage(Of DataSourceModel)

        Public Sub New(ByVal view As IChooseDataSourceNamePageView, ByVal dataSourceNameCreator As IDataSourceNameCreationService)
            MyBase.New(view, dataSourceNameCreator)
        End Sub
        Public Overrides Function GetNextPageType() As Type
            Return GetType(CustomChooseDataSourceTypePage)
        End Function
    End Class
End Namespace
